.puprc

```diff
@@ -1,28 +1,19 @@
 {
 	"build": [
-		"cd common && rm -rf vendor",
-		"cd common && git checkout -- vendor",
-		"cd common && composer install --no-dev",
-		"cd common && npm run build",
+		"cd common && pup build",
 		"composer install --no-dev",
 		"npm ci",
 		"npm run build"
 	],
 	"build_dev": [
-		"cd common && rm -rf vendor",
-		"cd common && git checkout -- vendor",
-		"cd common && composer install --no-dev",
-		"cd common && npm run build",
+		"cd common && pup build",
 		"composer install",
 		"npm ci",
 		"npm run build"
 	],
 	"checks": {
 		"tbd": {
-			"dirs": [
-				"src",
-				"common/src"
-			]
+			"dirs": [ "src" ]
```

Comment:
I'd undo this change, so that when packaging TEC it can control all TBD detection in both itself and common.